src/context/AppProvider/AppContext.tsx

```diff
@@ -63,6 +64,8 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
   useTransactionsSubscriber()
   // App-wide long-poll of limit orders
   useLimitOrders()
+  // App-wide long-poll of notifications
+  useTradeActionSubscriber()
```

Comment:
q: Isn't the intent of the system to have a global "notification" / "action" system with a bunch of different pollers that plug in? Shouldn't we at the very least have a master action hook?